Dev content review (#15)
OpenLiberty/guide-microprofile-config-profile: README.adoc

include::{common-includes}/gitclone.adoc[]

// =================================================================================================

> == Trying the basic configuration

Reviewer: though grammatically correct, this title feels sort of vague, You might consider something more specific, like 
"Accessing the system service in the testing environment" or "Testing the default configuration"

Author: changed to "Starting from the default configuration"

> Run the `query` service in development mode with the `system` service that is running in the testing enivronment.

Reviewer: you also might revise this sentence so it reflects the order of the tasks in this section, like

```suggestion
Deploy the `system` service in the testing environment and access it with the `query` service, which is running in dev mode.
```

```

// =================================================================================================
=== Switching the system service to development mode

> Assume that the `system` service is required to fix something. The `query` service need connect to its development environment instead of testing environment.

Reviewer: ```suggestion
Assume that the `system` service is needs maintenance. The `query` service must now connect to it in the development environment instead of the testing environment.
```

* [hotspot=testUnknownHost file=0]`testUnknownHost()` verifies that an unknown host or a host that does not expose their JVM system properties is correctly handled with a fail message.

> === Running the tests in dev mode

Reviewer: ```suggestion
=== Running the tests in the development environment
```

 To avoid confusion between dev mode and development environment. Although tests are run in dev mode here, the contrast with the next section is between environments development vs test.

include::finish/scripts/testApp.sh[]
----

> === Running the tests in testing environment

Reviewer: ```suggestion
=== Running the tests in the testing environment
```
